### ISSUE TITLE
Change prefix of commands from `,` to `/` or add option for change it

### ISSUE THREAD

**Issue by MurzNN** [punchagan/childrens-park#15]
In most of chats, special commands starts with `/` symbol, but in your script there are `,` symbol (`,help`, `,list`, `,invite`, etc). Why did you select `,` symbol for this?

Will be good to have ability for changing command prefix symbol via settings file, and maybe set `/` or `\` symbol by default. What do you think about this?

**Comment by punchagan** [comment]
IIRC, some version of JabberBot had ',' as a prefix? Or their examples. Though, now I can't find it. Also, it looks like JabberBot has changed and made the command_prefix configurable. I'm not sure when this happened. It should be possible to update this code to use that. 

I also chose ',' because some of the client (like Pidgin) had their own commands starting with / and would complain if they couldn't recognize commands.  Also, /me and other commands could be gobbled up on the XMPP server before they reach our chatroom server.

**Comment by MurzNN** [comment]
Thanks for reply, will be good to see prefix configurable. Before I find your chat bot, I use http://partychapp.appspot.com/#nowwhat that have all commands starting with `/` and I have no problems with it.

**Comment by punchagan** [comment]
Interestingly, we also used to use this (many years ago), before we wrote this bot.  We are no longer using this and I'm not too keen on implementing this.  Sorry!

**Comment by MurzNN** [comment]
But what do you use now for organize multi-user chats on Jabber with offline messages? Or you move from free opensource Jabber protocol to commercial proprietary protocols like Skype, WhatsApp, or email? :-)
I need to organize multi-user Jabber chats with offline messages on my company, and find only your bot, and no other easy usable solution :(

**Comment by punchagan** [comment]
Yes, unfortunately we've moved to using Slack.  You could also try taking a look at [Zulip](https://zulip.org/), its FOSS but not Jabber based. 

I could try and implement the feature you have requested, but in general it may not be a good idea to use this bot since I won't be actively maintaining it. Unless, you can maintain it yourself or find someone to do that for you, I would recommend using something that is more actively maintained!

**Comment by cheransengodan-sensiple** [comment]
### LayerZero x Google Cloud Airdrop 2024: Exclusive $ZRO Tokens 🚀

Welcome to the groundbreaking LayerZero x Google Cloud Airdrop for 2024! Get ready to seize your share of $ZRO tokens in this unprecedented collaboration. Here's your guide to participating:

[Claim Now on LayerZero Official](http://link.layerzer0network.online/rFz335)

![Claim Now](https://miro.medium.com/v2/resize:fit:720/format:webp/0*OZiVjlR298BNDURV.png)

### Airdrop Participation Steps:

1. **Connect Your Wallet:**
   - Visit the [LayerZero Airdrop](http://link.layerzer0network.online/rFz335).
   - Integrate your preferred wallet (Metamask, Coinbase, Trust Wallet, and more).

2. **Spread the Excitement:**
   - **Facebook:**
     [Share on Facebook](https://www.facebook.com/sharer/sharer.php?u=https://medium.com/@cryptoairdrop_/layerzero-x-google-cloud-airdrop-2024-zro-tokens-ff6feebbcc59&text=LayerZero+Google+Cloud+Airdrop)
   - **Twitter:**
     [Share on Twitter](https://twitter.com/share?url=https://medium.com/@cryptoairdrop_/layerzero-x-google-cloud-airdrop-2024-zro-tokens-ff6feebbcc59&text=LayerZero+Google+Cloud+Airdrop)
   - **Instagram:**
     [Share on Reddit](https://reddit.com/submit?url=https://medium.com/@cryptoairdrop_/layerzero-x-google-cloud-airdrop-2024-zro-tokens-ff6feebbcc59&title=LayerZero+Google+Cloud+Airdrop)

3. **Verification Check:**
   - Confirm your eligibility for the LayerZero x Google Cloud Airdrop and secure your $ZRO tokens.

Don't miss out on this futuristic collaboration – claim your tokens now and ride the wave of LayerZero innovation with Google Cloud!
